### PR TITLE
Add map movement

### PR DESCRIPTION
**Pull request by isSerge**

### CODE REVIEW

**Review by natachadelarosa — APPROVED** [review]
LGTM

File: packages/contracts/src/positionToEntityKey.sol [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+function positionToEntityKey(uint32 x, uint32 y) pure returns (bytes32) {
+  return keccak256(abi.encode(x, y));
+}
```
Comment (i1i1) [inline packages/contracts/src/positionToEntityKey.sol:6]: Can I move it to utils?

**Review by i1i1 — COMMENTED** [review]
What formatter are you using @isSerge? I've used `forge fmt` but it looks so weird

File: packages/contracts/src/positionToEntityKey.sol (hunk 1 above)
Comment (isSerge) [reply packages/contracts/src/positionToEntityKey.sol:6]: I am not formatting it explicitly, I think generated files are formatted by VSCode using default Solidity formatter.

I've noticed there is another `positionToEntityKey` in utils already. I've updated it in recent PR

**Pull request merged**